Common/Utility/DocDbRestUtility.cs

```diff
@@ -267,6 +270,26 @@ public DocDbRestUtility(IConfigurationProvider configProvider)
             return JObject.Parse(response);
         }
 
+        public async Task<DeviceND> SaveNewDocumentAsyncND(DeviceND document)
```

Comment:
Actually, only you have changed it, as far as I can see from the history of  DocDbRestUtility and IDocDbRestUtility, so can you create ND versions of it ?